### PR TITLE
Group B: Tom Kennon

### PR DESCRIPTION
**Pull request by tomkennon**
1st commit

### CODE REVIEW

File: _posts/2020-06-05-tom.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+date: 2020-06-05
+---
+
+
+This summer I'm looking forward to sports being back.
+I recommend you read/watch Rick and Morty.
+I've picked up running again.
+I'd like to visit Florida
+
+![Look at me!](https://www.google.com/imgres?imgurl=https%3A%2F%2Fi.ytimg.com%2Fvi%2FqUYvIAP3qQk%2Fmaxresdefault.jpg&imgrefurl=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DqUYvIAP3qQk&tbnid=dobK4PISFjRZQM&vet=12ahUKEwjp4qGu7OrpAhXOC98KHTgoCxAQMygCegUIARDfAQ..i&docid=bIa4_gYzbZa-3M&w=1280&h=720&q=rick%20and%20morty%20meeseegs&safe=strict&ved=2ahUKEwjp4qGu7OrpAhXOC98KHTgoCxAQMygCegUIARDfAQ)
```
Comment (zach004) [inline _posts/2020-06-05-tom.md]: Image link is broken, please resolve

Comment (tomkennon) [reply _posts/2020-06-05-tom.md]: fixed it, ty

File: _posts/2020-06-05-tom.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+This summer I'm looking forward to sports being back.
+I recommend you read/watch Rick and Morty.
+I've picked up running again.
+I'd like to visit Florida
+
+![Look at me!](https://vignette.wikia.nocookie.net/rickandmorty/images/6/6c/MeeseeksHQ.png/revision/latest?cb=20150930232412)
+
+Comment your reply here.
```
Comment (zambrip) [inline _posts/2020-06-05-tom.md:15]:
```rick and morty has been weird recently
```

**Review by zambrip — APPROVED** [review]

**Comment by MajesticUnic0rn** [comment]
that profile pic suit looks too big on you

**Comment by tomkennon** [comment]
> that profile pic suit looks too big on you

brutal my guy

File: _posts/2020-06-05-tom.md (hunk 2 above)
Comment (asingh9) [inline _posts/2020-06-05-tom.md:15]:
```suggestion
The new season is hilarious```

**Pull request merged**